library/Zend/Mail/Storage/Part.php

```diff
@@ -377,9 +385,12 @@ public function __get($name)
      */
     public function __isset($name)
     {
+        $headers = $this->getHeaders();
+        if (empty($headers) || !is_object($headers)) {
```

Comment:
getHeaders() always returns headers container instance. This change is not needed.